aspnet/getting-started/installing-on-mac.rst

```diff
@@ -1,21 +1,21 @@
 Installing ASP.NET 5 On Mac OS X
 ================================
 
-By `Steve Smith`_, `Daniel Roth`_
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_
 
-.. contents:: In this article
-  :local:
-  :depth: 1
+.. contents:: Sections
+ :local:
+ :depth: 1
 
 Install ASP.NET 5 with Visual Studio Code
 -----------------------------------------
 
 The easiest way to get started building applications with ASP.NET 5 is to install the latest version of Visual Studio Code.
 
-1. Install `Visual Studio Code <https://go.microsoft.com/fwlink/?LinkID=534107>`__
+#. `Update OS X for Visual Studio Code <https://code.visualstudio.com/docs/supporting/howtoupdate#_updating-on-os-x>`__
```

Comment:
Most people on OS X won't have Visual Studio Code at all. I would leave the original text (although the fwlink should be to 534106 - this was fixed recently).